### DISCUSSION TITLE
Livewire Full-Page Components Occasionally Display Blank Pages After Production Deployment

### DISCUSSION THREAD

**Discussion by DeividasVarnagisOrbio** [livewire/livewire#5679] [1. Help]
The issue arises sometimes following server deployment, and its inconsistent nature poses a challenge for reproduction. Unfortunately, server logs have not provided any error messages or additional information to aid in resolving the problem.

It is important to note that other Livewire components render successfully, indicating that the issue is specific to full-page components.

Upon further examination of the response, we observed a blank HTML page with a 200 status code. The rendered view appears to be entirely empty, with neither layout nor content displayed. Additionally, no JavaScript errors have been detected in the browser's developer tools.

We are currently exploring potential causes, including deployment-related complications or view caching.

Any help is appreciated!!!

Environment:
- Livewire 2.11.2
- Laravel 9.52.5
- Vite 4.2.1

The given code is used to render the full-page component in the app.
![image](https://user-images.githubusercontent.com/102963835/236200288-065f17de-6525-4155-9ff0-9045f9a0c80c.png)

Happened again - adding displayed HTML:

![image](https://github.com/livewire/livewire/assets/102963835/14576c0c-03bb-4195-bf0a-7ba2676676f1)

**Comment by michael-rubel** [comment]
@DeividasVarnagisOrbio do you use Laravel Octane in your app? I once faced a very similar issue, but that was about two years ago when Octane was released and I added it to the Livewire app.

**Reply by DeividasVarnagisOrbio** [reply]
I appreciate your suggestion but our application does not use Laravel Octane.
